node-module-system/calculate.js

```diff
@@ -0,0 +1,17 @@
+import add from './add.js';
+import subtract from './subtract.js';
+import multiply from './multiply.js';
+import divide from './divide.js';
+
+const param1 = process.argv[2];
+const param2 = process.argv[4];
+
+if (process.argv[3] === 'plus') {
```

Comment:
You repeat `process.argv[3]` several times here. It would be nice to remove that repetition.